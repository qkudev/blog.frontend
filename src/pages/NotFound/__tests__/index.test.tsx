import * as React from 'react'
import renderer from 'react-test-renderer'
import { MemoryRouter } from 'react-router-dom'
import NotFoundPage from '../index'

describe('NotFoundPage', function() {
  test('Matches with snapshot', function() {
    const notFoundPage = renderer.create(
      <MemoryRouter>
        <NotFoundPage />
      </MemoryRouter>
    )
    let tree = notFoundPage.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
