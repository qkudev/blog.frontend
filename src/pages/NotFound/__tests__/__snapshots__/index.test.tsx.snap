// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage Matches with snapshot 1`] = `
<div
  className="page not-found"
>
  <div
    id="page-not-found"
  >
    <div
      className="display-1"
    >
      404
    </div>
    <h1>
      NOT FOUND
    </h1>
  </div>
  <a
    href="/"
    onClick={[Function]}
  >
    Back to homepage
  </a>
</div>
`;
